### ISSUE TITLE
Dire Maul - NORTH: Mobs pathing

### ISSUE THREAD

**Issue by nafk1450** [elysium-project/classic-bug-tracker#1136]
Since the server went down yesterday(4/20/2019) mobs in dire maul path through walls and floor, like the models and hitboxes of the enviroment didn't match.

**Comment by deartonym** [comment]
Yeah we met with exactly the same problem. And the King Gordok can intercept on the air! that is not acceptable, ridiculous.

**Comment by Continueur** [comment]
Yeah same issue, let’s hope they fix it soon!

**Comment by europoor** [comment]
A week later and still no one has been assigned to this issue...but thank god you fixed the easter event kappa

**Comment by Ziggy-QA** [comment]
This has been confirmed and forwarded to developers.
